### PR TITLE
#2487 space between add-on description and screenshots can be reduced

### PR DESCRIPTION
**Pull request by tsl143**
fix https://github.com/mozilla/addons-server/issues/2487

### CODE REVIEW

**Comment by tsl143** [comment]
@tofumatt , Consider the PR

**Comment by EnTeQuAk** [comment]
Can you maybe add a screenshot to show the intended behavior? This makes it easier to verify it later for QA on different systems.

**Comment by tofumatt** [comment]
@tsl143 Hi there. No need to ping us in the PR or add the comment, we get notifications once the PR is submitted 😄. If you could use the format "fixes #2487" in your description or commit message it will also automatically close the issue 👍

Could you submit a screenshot like @EnTeQuAk said? After that I can review this easier 😄

**Comment by tsl143** [comment]
Sure :)

after1
before1

**Comment by tofumatt** [comment]
Awesome, r+ 👍

File: static/css/restyle/restyle.less [hunk 1]
```diff
@@ -289,7 +289,7 @@ body:not(.home) .amo-header,
 .addon-details .header-bg,
 .home .header-bg {
   min-height: 130px;
-  margin-bottom: 10px;
+  margin-bottom: 0;
```
Comment (designakt) [inline static/css/restyle/restyle.less]: @tofumatt 
This pull request also impacted the homepage which it shouldn't and reduced the space between the blue and white area there as well.
The margin-bottom should only be 0 on .addon-details .header-bg, but not on .home .header-bg

Comment (tofumatt) [inline static/css/restyle/restyle.less]: Filed #2691

**Pull request merged**